constants.py

```diff
@@ -49,7 +49,7 @@
 SCALER_CONFIG_DEFAULT_SERVER_PRIVATE_IP = '10.0.0.4'
 SCALER_CONFIG_DEFAULT_SERVER_PORT = '33450'
 
-# Platform specific constants
+# Platform specific constants, respecting the EC2 Image Builder requirement of sentence casing
```

Comment:
suggest added a commented line below this with a link to https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-component.html